modules/checklist/assets/js/app/components/checklist-card-content.js

```diff
@@ -13,6 +13,7 @@ const ChecklistCardContent = ( { step, setSteps } ) => {
 		learn_more_url: learnMoreUrl,
 		learn_more_text: learnMoreText,
 		image_src: imageSrc,
+		is_locked: isLocked,
```

Comment:
Please remove

Reply:
Without this i get:  `Uncaught (in promise) ReferenceError: isLocked is not defined
    at _callee$ (checklist-card-content.js:32:94)`